Update v1.0.0rc1

jcannon-gplates opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Create _release candidate_ version 1.0.0rc1 on the `rc` branch of conda-forge feedstock.

jcannon-gplates committed:
Add pygplates_rc channel target to new rc branch.

jcannon-gplates committed:
Update to use scikit-build-core build backend.

jcannon-gplates committed:
Use gmp on Windows (not just unix) for conda.

This mirrors how 'cgal' references 'cgal-cpp'
- see conda-forge/cgal-feedstock@0f6f2ed

jcannon-gplates committed:
Add stdlib("c") to meta.yml.

jcannon-gplates committed:
Get source from release/pygplates-1.0 branch in GPlates GitHub.

jcannon-gplates commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.06.08.06.51

jcannon-gplates committed:
Get source from PyPI (instead of GitHub).

Using GitHub was giving the errors:
- git: 'lfs' is not a git command. See 'git --help'
- error: pathspec 'release/pygplates-1.0' did not match any file(s) known to git

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12196813238](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12196813238). Examine the logs at this URL for more detail.</sub>

jcannon-gplates committed:
Attempt to fix error that libgl is "not in reqs/run" (on Linux).

Using advice from [here](https://conda-forge.org/docs/maintainer/knowledge_base/#libgl).

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [51]


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12198680315](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12198680315). Examine the logs at this URL for more detail.</sub>

jcannon-gplates committed:
Attempt to fix error that libglx is "not in reqs/run" (on Linux).

Adding host req "libglx-devel".

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.
